### ISSUE TITLE
scroll position is still jerky when backfilling

### ISSUE THREAD

**Issue by matrixbot** [vector-im/vector-web#528]
Created by @ richvdh:matrix.org.

**Comment by richvdh** [comment]
Matthew is still observing jerky scroll position when we are backfilling: specifically, when doing a two-finger scroll backwards in time, the scroll position sometimes jumps to the top of the room rather than preserving position.

**Comment by richvdh** [comment]
The problem is that we see a scroll event arriving *after* we reset the scroll position, but with the scroll offset reset to old values:

```
2015-12-16 15:08:24.147 RoomView.js:325 paginate complete
2015-12-16 15:08:24.211 RoomView.js:926 Scrolled to event $145027813095505zwpiN:matrix.org + 21 : 1832
2015-12-16 15:08:24.237 RoomView.js:926 Scrolled to event $145027813095505zwpiN:matrix.org + 21 : 1832
2015-12-16 15:08:24.242 RoomView.js:395 Scroll event: offset now 528
2015-12-16 15:08:24.243 RoomView.js:399 Saved scroll state Object {atBottom: false, lastDisplayedEvent: "$145027622893037urerU:matrix.org", pixelOffset: 81}
```

- why is the scroll offset suddenly reset from 1832 to 528?

We could hack around this by ignoring the scroll events until we get one which (roughly?) matches the scroll offset we assigned... but that feels ugly and fragile. I'd like to understand what is causing the reset.

**Comment by richvdh** [comment]
Hopefully this has been fixed by matrix-org/matrix-react-sdk#56
